src/coreclr/nativeaot/System.Private.CoreLib/src/System/Reflection/RuntimeAssembly.cs

```diff
@@ -1,10 +1,38 @@
 // Licensed to the .NET Foundation under one or more agreements.
 // The .NET Foundation licenses this file to you under the MIT license.
 
+using System.Globalization;
+using System.Security;
+
+using Internal.Reflection.Augments;
+
 namespace System.Reflection
 {
     // Base class for runtime implemented Assembly
     public abstract class RuntimeAssembly : Assembly
     {
+#pragma warning disable IDE0060
+        [DynamicSecurityMethod] // Methods containing StackCrawlMark local var has to be marked DynamicSecurityMethod
```

Comment:
`StackCrawlMark` is not a thing in native AOT and `DynamicSecurityMethod` has no effect.